CIS1940.cabal

```
@@ -40,6 +42,9 @@ library
   ghc-options: -Wall -Wcompat -Widentities -Wincomplete-record-updates -Wincomplete-uni-patterns -Wmissing-export-lists -Wmissing-home-modules -Wpartial-fields -Wredundant-constraints
   build-depends:
       base >=4.7 && <5
+      , aeson >= 2.1.2.1
```

When using stack for developing applications, it's common remove version bounds and let the library versions be defined by stack resolver that the project is using (https://www.stackage.org/lts-22.43 in your case)
```suggestion
      , aeson
```


This is different from libraries (which you would publish on hackage for other people to use - those SHOULD have version bounds specified, to make it clear which dependency versions they are expected to work with).